torch_xla/csrc/tensor.cpp

```diff
@@ -1725,8 +1740,28 @@ XLATensor XLATensor::threshold_backward(const XLATensor& grad_output,
       grad_output.GetIrValue(), input.GetIrValue(), threshold));
 }
 
-XLATensor XLATensor::t(const XLATensor& input) {
-  return input.CreateFrom(ir::ops::TransposeOp(input.GetIrValue()));
+XLATensor XLATensor::transpose(const XLATensor& input, xla::int64 dim0,
+                               xla::int64 dim1) {
```

Comment:
Why not a TransposeOp in ops/, used in both places?

Reply:
Done.